src/views/Profile/components/ApproveConfirmButtons.tsx

```diff
@@ -67,6 +68,7 @@ const ApproveConfirmButtons: React.FC<ApproveConfirmButtonsProps> = ({
   onConfirm,
   buttonArrangement = ButtonArrangement.ROW,
   confirmLabel,
+  confirmId,
```

Comment:
`ApproveConfirmButtons` receive `confirmId` that is passed as an id to the `confirm` buttons